lib/std/c.zig

```diff
@@ -2206,6 +2206,39 @@ pub const SC = switch (native_os) {
     .linux => linux.SC,
     else => void,
 };
+
+pub const _SC = switch (native_os) {
```

Comment:
How come? The neighboring code uses structs. Did the convention change since it was written?

Reply:
Type safety. A lot of the surrounding code was written before non-exhaustive enums and packed structs were available in the language.

I wouldn't have blocked the PR for this, but since there were other things needing addressed, I included it in the review.